Makefile

```diff
@@ -1,34 +1,12 @@
-UNAME_M = $(shell uname -m)
-ARCH=
-ifeq ($(UNAME_M), x86_64)
-	ARCH=amd64
-else ifeq ($(UNAME_M), aarch64)
-	ARCH=arm64
-else 
-	ARCH=$(UNAME_M)
-endif
-
 ORG        ?= rancher
-TAG        ?= v1.19.0
-GO_VERSION ?= $(shell echo $(TAG) | sed -e "s/v\(.*\)b.*/\1/g")
+TAG        ?= v1.22.0b1
+GOLANG_VERSION ?= $(shell echo $(TAG) | sed -e "s/v\(.*\)b.*/\1/g")
+
 
-.PHONY: image-build
```

Comment:
I left the Makefile targets just in case somebody wanted to run them manually. We agreed with Derek that it might be useful

Reply:
except the `image-manifest` target which makes no sense without the file `manifest.tmpl`

Reply:
During early experimentation on this, I updated the image-manifest step to use the docker cli, so it should still be working